open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -38,14 +38,6 @@ the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-serve
  * Developers can specify the environment variable(s) in a .env file, by copying [.env_sample](nodejs/.env_sample) to a file
  called .env in the same folder and amend the values as required.
  * Once the .env file is in place, build and start the server using `npm start`. Use other npm parameters as specified in [package.json](nodejs/package.json).  
-
-## FOR NEW DEVELOPMENT (as of 8/2/2020)
- * Currently, the UI is not optimized to run in development mode. Developers must rebuild the static files every time a change is made.
- * Because of this, work is currently underway to remake the client with Create React App. This will be done iteratively, and updates posted here.
- * Old client code will be maintained until the cutover is complete, so developers can run the traditional way if desired.
- * The new client is maintained in the cra-client directory.
- * Set up your environment by running `yarn`.
```

Comment:
presumably from this directory - but also would require running yarn? might be worth adding a link even for this stage - though when we LATER do maven integration it shouldn't be needed (there we need to minimize any external dependencies)

Reply:
We are actually not using Yarn anymore. I had it in the first PR I had merged, but after discussing further with @davidradl we decided to stick with NPM instead and have removed all Yarn references.